client-extensions/ticket-batch/LCP.json

```diff
@@ -1,8 +1,5 @@
 {
 	"cpu": 0.2,
-	"dependencies": [
```

Comment:
What was the exact issue here @allen-ziegenfus ? Wondering if this is something we need to take a closer look into?